unified_planning/model/expression.py

```diff
@@ -318,13 +318,54 @@ def Forall(
             node_type=OperatorKind.FORALL, args=expressions, payload=vars
         )
 
+    def Always(self, *expression: BoolExpression) -> "up.model.fnode.FNode":
```

Comment:
Here, all the added methods must be documented with the format of the other expressions constructors.

Same in the `shortcuts.py` file. (All the changes done here must be done also in the shortcuts file).